src/server/login.js

```diff
@@ -197,11 +209,23 @@ module.exports = function (client, server, options) {
     pluginChannels(client, options)
 
     if (client.profileKeys) {
-      client.verifyMessage = (packet) => {
-        const signable = concat('i64', packet.salt, 'UUID', client.uuid, 'i64',
-          packet.timestamp, 'pstring', packet.message)
-
-        return crypto.verify('sha256WithRSAEncryption', signable, client.profileKeys.public, packet.crypto.signature)
+      client.verifyMessage = (packet, acknowledgements) => {
```

Comment:
dupe of server/chat.js ?

Reply:
Code in chat.js is for the chat session packet, which only exists in 1.19.3. This code is for 1.19/1.19.2